### DISCUSSION TITLE
中原師生展現研發創新實力 「全自動AI垃圾分類系統」奪金

### DISCUSSION THREAD

**Discussion by awehao** [1101-GS3073-C/Python-Community-GS3073-C#574] [科技新知]
我覺得這個好方便歐，可以讓世界更乾淨诶，垃圾沒分類確實造成了很多人的困擾
在座的各位要確實做垃圾分類阿，不要連這麼簡單的事都被ai取代
垃圾分類從你我做起，各位衝啊!
年輕的一代，可以從更單純世界，也因此可以找到更多問題，隨著科技的進步，解決方法更加多元更加多元，垃圾分類系統看似簡單，但確實能帶來很大的幫助
https://tw.news.yahoo.com/%E4%B8%AD%E5%8E%9F%E5%B8%AB%E7%94%9F%E5%B1%95%E7%8F%BE%E7%A0%94%E7%99%BC%E5%89%B5%E6%96%B0%E5%AF%A6%E5%8A%9B-%E5%85%A8%E8%87%AA%E5%8B%95ai%E5%9E%83%E5%9C%BE%E5%88%86%E9%A1%9E%E7%B3%BB%E7%B5%B1-%E5%A5%AA%E9%87%91-034600021.html?guccounter=1&guce_referrer=aHR0cHM6Ly93d3cuZ29vZ2xlLmNvbS8&guce_referrer_sig=AQAAAGrRLux9-wmvi2BzCHQtTYd_R1oS0r7cdSFB1VfNuM8BRwlaHeWEQoSuEhugjYjZxbpLzLuDeVVmw9Zqnz5kryCVS3DzW_c76FTHgjFjL9gct07W-0kk1uyuZzQ96-NGcCtn9YbedrIlIcrCDOpPNappKEz2gtQQtrTkuLBjr7y2

**Comment by number32984** [comment]
資源回收確實重要，但人們無法做到就需靠ai的協助來幫助改善垃圾的問題

**Comment by skyling0299** [comment]
好有趣，如果能加上自動辨識和機器手臂，或許可以解決更多問題! 感覺很棒呢!

**Comment by Raywei0214** [comment]
每個人的一生中，幾乎可說碰到自動AI垃圾分類裝置這件事，是必然會發生的。那麼，如果此時我們選擇忽略自動AI垃圾分類裝置，那後果可想而知。

**Comment by larry40305** [comment]
以後人類都不用分類，全部都丟給AI做
